assets/scripts/im.pontoWikiaAppsBridge.js

```diff
@@ -27,8 +26,9 @@ define('im.pontoWikiaAppsBridge', ['ponto', 'im.window'], function (ponto, w) {
 			var target = event.target;
 
 			if (
-				target.tagName === 'A' &&
-				target.className.indexOf(poiArticleLinkClass) !== -1
+				target.tagName === 'A' || target.tagName === 'IMG' &&
+				target.className.indexOf(config.articleLinkClassName) !== -1 ||
+				target.parentNode.className.indexOf(config.articleLinkClassName) !== -1
```

Comment:
The same here, `&&` has a higher precedence than `||`
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence#Table